packages/monaco/src/browser/monaco-editor.ts

```diff
@@ -357,6 +360,7 @@ export class MonacoEditor extends MonacoEditorServices implements TextEditor {
         if (this.node) {
             const layoutSize = this.computeLayoutSize(this.node, dimension);
             this.editor.layout(layoutSize);
+            this.onResizeEmitter.fire(undefined);
```

Comment:
Why fire this event here if it's always going to be `undefined`?

Reply:
Another good catch, I had intended for the `undefined` case to be handled by forwarding the request to the editor's `resizeToFit` method but hadn't handled it properly in the setter. This should be fixed now.